ani-cli.1

```diff
@@ -0,0 +1,101 @@
+.\" Automatically generated by Pandoc 2.9.2.1
+.\"
+.TH "ANI-CLI (1) ani-cli" "" "\[lq]June 19 2022\[rq]" "" ""
+.hy
+.SH NAME
+.PP
+ani-cli - watch anime from the commandline
+.SH SYNOPSIS
+.PP
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] [-a ] [-d | -p ] []
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] [-s] [-v] [-q ] -c
+.PD 0
+.P
+.PD
+\f[B]ani-cli\f[R] -h | -D | -U | -V
+.SH DESCRIPTION
+.PP
+\f[B]ani-cli\f[R] is a shell script to browse and search anime from the
+command-line.
+This tool scrapes the site animixplay.
+\f[B]ani-cli\f[R] without options defaults to the mpv media player.
+.SH OPTIONS
+.TP
+\f[B]-c\f[R]
+continue watching anime from history.
+.TP
+\f[B]-a\f[R] \[dq] episode\[dq]
+specify episode to watch.
+.TP
+\f[B]-h\f[R]
+Show summary of options.
+.TP
+\f[B]-d\f[R]
+download episode.
+.TP
+\f[B]-p\f[R] \[lq]download_dir\[rq]
+download episode to specified directory.
+.TP
+\f[B]-q\f[R] \[lq]quality\[rq]
+set video quality (best|worst|360|480|720|1080).
```

Comment:
The short descriptions for a, p and q are a formatting nightmare and since no other argument has them, we should just drop them for good.
![image](https://user-images.githubusercontent.com/66648475/208686629-4cf1e99f-dd64-416e-8b82-817565e0bb04.png)

Reply:
IMO a better description is better than dropping them. Especially `-a` since it is useful and can easily be confused with `-e` (which doesn't exist but resembles more the word "episode").
I rewrote the OPTIONS (following alphabetical order the same way `man ls` and `man mkdir` are laid out) and removed duplicate options.
![image](https://user-images.githubusercontent.com/74317212/208939833-0594935a-4c73-44f3-b3a9-b74322a058f2.png)

I hope you reconsider wanting to drop them.